katharsis-meta/src/main/java/io/katharsis/meta/MetaModule.java

```diff
@@ -8,7 +8,7 @@
 import io.katharsis.core.internal.utils.PreconditionUtil;
 import io.katharsis.legacy.registry.DefaultResourceInformationBuilderContext;
 import io.katharsis.meta.internal.MetaRelationshipRepository;
-import io.katharsis.meta.internal.MetaResourceRepository;
+import io.katharsis.meta.internal.MetaResourceRepositoryImpl;
```

Comment:
Can we rename this to `DefaultMetaResourceRespository` ? an `impl` tells me nothing about the class. But a `Default...` tells me it's the default implementation